### PR TITLE
update installing nightly snaphost flow

### PR DESCRIPTION
**Pull request by sandrobonazzola**
Changes proposed in this pull request:

- Update the instructions for installing oVirt master snapshot

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @sandrobonazzola 

This pull request needs review by:  @tinez @sanjacodes @michalskrivanek @ahadas @shenitzky @mwperina

### CODE REVIEW

File: source/develop/developer-guide/vdsm/developers.md [hunk 1]
```diff
@@ -37,27 +37,7 @@ authors:
...
-      yum install http://resources.ovirt.org/pub/yum-repo/ovirt-release40.rpm 
-      yum install http://resources.ovirt.org/pub/yum-repo/ovirt-release36.rpm 
-      yum install http://resources.ovirt.org/pub/yum-repo/ovirt-release35.rpm
-      
- for Fedora 
-    
-     dnf install http://resources.ovirt.org/pub/yum-repo/ovirt-release40.rpm 
-     dnf install http://resources.ovirt.org/pub/yum-repo/ovirt-release36.rpm 
-     dnf install http://resources.ovirt.org/pub/yum-repo/ovirt-release35.rpm
+To build VDSM, enable the oVirt repositories by following the [Install nightly snapshot](develop/dev-process/install-nightly-snapshot.html) guide
```
Comment (sanjacodes) [inline source/develop/developer-guide/vdsm/developers.md:40]: `.` at the end missing

File: source/develop/developer-guide/vdsm/installing-vdsm-from-rpm.md [hunk 2]
```diff
@@ -28,9 +28,7 @@ execute the following command as root:
 
 ### Configuring oVirt's Repository
 
-Execute the following command:
-
-      sudo yum install http://resources.ovirt.org/releases/ovirt-release/ovirt-release-master.rpm
+Follow the [Install nightly snapshot](develop/dev-process/install-nightly-snapshot.html) guide
```
Comment (sanjacodes) [inline source/develop/developer-guide/vdsm/installing-vdsm-from-rpm.md:31]: `.`

**Review by sanjacodes — APPROVED** [review]
Computer says no - see CI.

Would add some missing `.`s and maybe fix the varying capitalization in the headings, but we can do an extra PR for that after this one.

File: source/develop/developer-guide/vdsm/developers.md (hunk 1 above)
Comment (sandrobonazzola) [reply source/develop/developer-guide/vdsm/developers.md]: done

**Review by sanjacodes — APPROVED** [review]

**Pull request merged**